owncloudApp/src/main/java/com/owncloud/android/db/ProviderMeta.java

```diff
@@ -33,11 +33,13 @@
 public class ProviderMeta {
 
     public static final String DB_NAME = "filelist";
-    public static final int DB_VERSION = 25;
+    public static final String NEW_DB_NAME = "owncloud_database";
+    public static final int DB_VERSION = 26;
```

Comment:
why don't you use the same version for room?

Reply:
Done